src/modules/sign/components/Signin.css

```diff
@@ -1,127 +1,94 @@
-.svg_logo{
+.container {
   display: flex;
-  margin: auto;
-  position: absolute;
-  width: 211px;
-  height: 33px;
-  left: 144px;
-  top: 40px;
-
+  flex-direction: column;
+}
+.svg_logo{
+  margin-left: 9rem;
+  margin-top: 2rem;
+  width: 11rem;
+  height: 2rem;
 }
 .centered_div {
-  position: absolute;
-  width: 29rem;
-  height: 28rem;
-  top: 45%;
-  left: 53%;
-  -webkit-transform: translate(-50%, -50%);
-  -moz-transform: translate(-50%, -50%);
-  -o-transform: translate(-50%, -50%);
-  -ms-transform: translate(-50%, -50%);
-  transform: translate(-50%, -50%);
-}
-.mid_centered_div {
-  padding: 2rem;
-  font-style: normal;
-  align-items: center;
   text-align: center;
-  color: #060553;
-  width: 22rem;
+  margin: 0 auto;
 }
 .sign_in_text {
   font-family: Inter, serif;
   font-weight: 500;
-  font-size: 24px;
-  line-height: 33px;
-  margin-top: 1rem;
+  font-size: 1.25rem;
+  line-height: 1rem;
+  margin-top: 5rem;
   color: #37464F;
 }
 .more_text{
   font-family: Inter, serif;
-  line-height: 22px;
-  font-size: 16px;
+  line-height: 1rem;
+  font-size: 0.75rem;
   font-weight: 500;
   color: #37464F;
 }
-
-.bottom_div {
-  position: absolute;
-  width: 349px;
-  left: 35rem;
-  top: 34rem;
-}
 .bottom_text {
   font-family: Inter, serif;
-  line-height: 19px;
-  font-size: 16px;
+  line-height: 1rem;
+  font-size: 0.65rem;
```

Comment:
I see that this value was `16px` before conversion to rem. By default, `16px` == `1rem`. Can you check again to be sure that all conversions are done correctly? You can use this online tool to do the conversions [PX to REM converter](https://nekocalc.com/px-to-rem-converter).